edgexfoundry/go-mod-bootstrap
# fix: Check nil pointer in `buildPaths()`

This PR will resolve #494 







If your build fails due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/go-mod-bootstrap/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions

Build and install edegfxoundry using go mod bootstrapper from this PR
```
snapcraft
snap install ./edgexfoundry_3.0.0-dev.114_amd64.snap --dangerous
```
Set env variables:
```
sudo snap set edgexfoundry apps.core-data.config.edgex-config-provider='none'
sudo snap set edgexfoundry apps.core-data.config.edgex-common-config=./config/core-common-config-bootstrapper/res/configuration.yaml
sudo snap restart edgexfoundry.core-data
```
Check logs:
```
$ journlctl -f | grep core-data
Mär 28 10:05:43 ubuntu edgexfoundry.core-data[271774]: Invalid value (<nil>) for key "Clients". Skipping. Invalid value (<nil>) for key "Tags". Skipping."level=INFO ts=2023-03-28T08:05:43.543578388Z app=core-data source=config.go:192 msg="Common configuration loaded from file with 0 overrides applied"
...
Mär 28 10:05:43 ubuntu edgexfoundry.core-data[271774]: level=INFO ts=2023-03-28T08:05:43.572789735Z app=core-data source=httpserver.go:133 msg="Web server starting (localhost:59880)"
Mär 28 10:05:43 ubuntu edgexfoundry.core-data[271774]: level=INFO ts=2023-03-28T08:05:43.572801546Z app=core-data source=message.go:50 msg="Service dependencies resolved..."
Mär 28 10:05:43 ubuntu edgexfoundry.core-data[271774]: level=INFO ts=2023-03-28T08:05:43.572805912Z app=core-data source=message.go:51 msg="Starting core-data 3.0.0-dev.114 "
Mär 28 10:05:43 ubuntu edgexfoundry.core-data[271774]: level=INFO ts=2023-03-28T08:05:43.572809113Z app=core-data source=message.go:55 msg="This is the Core Data Microservice"
Mär 28 10:05:43 ubuntu edgexfoundry.core-data[271774]: level=INFO ts=2023-03-28T08:05:43.572812418Z app=core-data source=message.go:58 msg="Service started in: 33.37508ms
```
## New Dependency Instructions (If applicable)
